packages/react-internal/etc/react-internal.api.md

```diff
@@ -7243,23 +7243,24 @@ export const useSlider: (props: ISliderProps, ref: React.Ref<HTMLDivElement>) =>
         tabIndex: number | undefined;
         'data-is-focusable': boolean;
         defaultChecked?: boolean | undefined;
-        defaultValue?: string | number | readonly string[] | undefined;
+        defaultValue?: string | string[] | undefined;
```

Comment:
This is a breaking change? If that's intentional, make sure this is captured in the v8 release notes.